src/styles/abstracts/_motion.scss

```
@@ -0,0 +1,56 @@
+// fast: hover 등 즉각 피드백 / base: 모달·토스트 등장 / slow: 스크롤 등장(reveal)
+$duration-fast: 150ms;
+$duration-base: 250ms;
+$duration-slow: 400ms;
+
+// 감속 곡선 하나만 쓴다. 등장·퇴장이 같은 커브라야 한 시스템으로 느껴진다.
+$ease-out: cubic-bezier(0.16, 1, 0.3, 1);
+
+// 스크롤 등장(reveal) 공통 스타일. `useScrollReveal` 훅이 붙이는 `data-revealed`와 짝을 이룬다.
+// 훅이 못 도는 환경(JS 실패·jsdom)이나 동작 줄이기 사용자를 위해 CSS 쪽에서도
+// prefers-reduced-motion이면 처음부터 드러난 상태로 둔다 — 콘텐츠가 영영 안 보이는 일은 없어야 한다.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)